tests/commands/test_build.py

```diff
@@ -404,13 +406,17 @@ def test_build_basic_complete_structure(tmp_path):
     assert zf.read('lib/ops/stuff.txt') == b"ops stuff"
 
 
-def test_build_code_simple(tmp_path):
+@pytest.mark.parametrize('has_config', [False, True], ids=lambda val: 'has_config={}'.format(val))
```

Comment:
We try to keep the individual tests as simple as possible, so to test the "there's a config" case, please create another test (which would be very similar to this one, but only will check that the config file is included).

Reply:
My own preference is for reusing the code instead of copying it, but I don't mind conforming to local practice.